### ISSUE TITLE
Unable to focus outside modeless modal

### ISSUE THREAD

**Issue by PorterNan** [microsoft/fluentui#18924]
### Environment Information
Package version(s): 8.17.4 and Latest version
Browser and OS versions: Chrome/Firefox/Edge on Windows
Describe the issue:
Go to the Modal documentation page
Click on Open Modal for the modeless modal
Hit Tab key multiple times

### Actual behavior:
Focus stays on close button

### Expected behavior:
Focus goes to the close button once, and then goes to the background content.

### Documentation describing expected behavior
This is a known issue:
https://github.com/microsoft/fluentui/issues/16179#issue-759487398

But looks like it is staying for too long and auto closed, other than add a hacky js code work around to force the focus outside of the modal, is there any other solution like disable the FocusTrapZone of the modeless modal?

Would like to know about any update of this bug as well

**Comment by ling1726** [comment]
The focus can only be escaped by clicking outside the modeless Modal with the mouse. Tab presses will still be trapped. This is definitely a bug. @ecraig12345  did you manage to follow up from #16179

@smhigley  Since it's possible to escape focus with mouse and not keyboard this should be an a11y bug too correct ?

**Comment by alcail** [comment]
Regarding this issue, or team just got an a11y bug filled out about keyboard trap.
This indeed happens on Modal component
MAS recommendation is to have no keyboard trap: https://liquid.microsoft.com/Web/Object/Read/MS.Accessibility/Requirements/02.01.02

**Comment by bsunderhus** [comment]
Seems like `focusInsideTrap={false}` refers more about keeping the focus inside `FocusTrapZone` but allowing focus to leak between `FocusZone` components instead of letting it leak outside of `FocusTrapZone`

@ecraig12345 , do we have any follow up on that?

**Comment by JamesBurnside** [comment]
Adding some information to this issue. It is indeed the [FocusTrapZone inside the modal](https://github.com/microsoft/fluentui/blob/b7f17e976f9e058f39c9fce4f0f9bb6eb4dfa577/packages/react/src/components/Modal/Modal.base.tsx#L414) that traps the keyboard focus.

One option that would benefit our Team that is blocked on this is to expose the `disabled` prop of the FocusTrapZone through the ModalProps. Allowing us to disable the focus trap zone would allow developers to circumvent this a11y issue.

**Comment by anjulgarg** [comment]
Do we have any update on this issue?

**Comment by tonyhallett** [comment]
https://github.com/microsoft/fluentui/commit/921725e6051089c90a076208c10af5d93ec26cc5
